packages/acceptance-tests/pkg-tests-specs/sources/node-modules.test.ts

```diff
@@ -1,4 +1,8 @@
 import {xfs, npath, PortablePath, ppath, Filename} from '@yarnpkg/fslib';
+import {exec}                                      from 'node:child_process';
+import {promisify}                                 from 'node:util';
+
+const execPromise = promisify(exec);
```

Comment:
Is this already present in a helper lib somewhere?

Reply:
We have a portable shell (`@yarnpkg/shell`) but since this is for a Windows specific test it's fine to use `exec` like this.